inc/article-structure-data/class-vk-article-structure-data.php

```
@@ -186,7 +186,13 @@ public static function get_author_array( $author_id = '' ) {
 
 		if ( ! $author_id ) {
 			// 表示中のページの投稿オブジェクトからユーザーIDを取得
+			// Get the user ID from the post object of the page being displayed.
 			global $post;
+			// global $post が null だったり post_author を持たない文脈では Warning を出さずに早期 return する。
+			// Bail out without raising a warning when global $post is null or has no post_author in the current context.
+			if ( ! isset( $post->post_author ) ) {
+				return;
```

PHPStan (2.2.1)

[error] Method VK_Article_Srtuctured_Data::get_author_array() should return array but empty return statement found.


(return.empty)